src/site/stages/build/drupal/graphql/taxonomy-fragments/GetTaxonomies.graphql.js

```diff
@@ -7,13 +7,15 @@ module.exports = `
           path
         }
         name
+        fieldAudienceRsHomepage
```

Comment:
This is a bool? 🙂

Reply:
Would this be used at the template layer or in metalsmith's plugins?

Reply:
Yeah `true` or `null` from what I am seeing